Tried in a Codespace, not working yet 😔 

```
$ ./gradlew build

> Configure project :build-logic
This version of Gradle expects version '4.0.14' of the `kotlin-dsl` plugin but version '4.1.0' has been applied to project ':build-logic'. Let Gradle control the version of `kotlin-dsl` by removing any explicit `kotlin-dsl` version constraints from your build logic.
WARNING: Unsupported Kotlin plugin version.
The `embedded-kotlin` and `kotlin-dsl` plugins rely on features of Kotlin `1.8.20` that might work differently than in the requested version `1.9.0`.

FAILURE: Build failed with an exception.

* Where:
Build file '/workspaces/Health/build.gradle' line: 25

* What went wrong:
An exception occurred applying plugin request [id: 'terasology-module']
> Failed to apply plugin 'terasology-metrics'.
   > Task with path ':extractConfig' not found in root project 'Health'.
```

--- a/build-logic/src/main/kotlin/terasology-metrics.gradle.kts
+++ b/build-logic/src/main/kotlin/terasology-metrics.gradle.kts
@@ -60,16 +60,28 @@ tasks.withType<Test> {
     }
 }
 
-tasks.withType<Checkstyle> {
-    dependsOn(tasks.getByPath(":extractConfig"))
-}
+ tasks.withType<Checkstyle> {
+    //FIXME: This is a workaround for module harness builds, where the config 
+    //       files are part of the harness but the task is not defined.
+    if (rootProject.name == "Terasology") {
+        dependsOn(tasks.getByPath(":extractConfig"))
+    }
+ }
 
 tasks.withType<Pmd> {
-    dependsOn(tasks.getByPath(":extractConfig"))
+    //FIXME: This is a workaround for module harness builds, where the config 
+    //       files are part of the harness but the task is not defined.
+    if (rootProject.name == "Terasology") {
+        dependsOn(tasks.getByPath(":extractConfig"))
+    }
 }
 
 tasks.withType<SpotBugsTask> {
-    dependsOn(tasks.getByPath(":extractConfig"))
+    //FIXME: This is a workaround for module harness builds, where the config 
+    //       files are part of the harness but the task is not defined.
+    if (rootProject.name == "Terasology") {
+        dependsOn(tasks.getByPath(":extractConfig"))
+    }
 }
 
 // The config files here work in both a multi-project workspace (IDEs, running from source) and for solo module builds

--- a/build.gradle
+++ b/build.gradle
@@ -332,4 +332,3 @@ task assembleBuildHarness(type: Zip) {
     // set the archive name
     archiveFileName.set('build-harness.zip')
 }
-    

